Documentation/operations/troubleshooting_clustermesh.rst

```diff
@@ -27,11 +27,51 @@ Generic
 Manual Verification of Setup
 ----------------------------
 
+ #. Validate that each cluster is assigned a **unique** human-readable name as well
+    as a numeric cluster ID (1-255).
+
+ #. Validate that the ClusterMesh apiserver is initialized correctly
+
+    .. code-block:: shell-session
+
+        $ kubectl logs -n kube-system deployment/clustermesh-apiserver -c apiserver
+        ...
+        level=info msg="Connecting to etcd server..." config=/var/lib/cilium/etcd-config.yaml endpoints="[https://127.0.0.1:2379]" subsys=kvstore
+        level=info msg="Got lease ID 7c0281854b945c05" subsys=kvstore
+        level=info msg="Got lock lease ID 7c0281854b945c07" subsys=kvstore
+        level=info msg="Initial etcd session established" config=/var/lib/cilium/etcd-config.yaml endpoints="[https://127.0.0.1:2379]" subsys=kvstore
+        level=info msg="Successfully verified version of etcd endpoint" config=/var/lib/cilium/etcd-config.yaml endpoints="[https://127.0.0.1:2379]" etcdEndpoint="https://127.0.0.1:2379" subsys=kvstore version=3.4.13
+
  #. Validate that the ClusterMesh subsystem is initialized by looking for a
-    ``cilium-agent`` log message like this::
+    ``cilium-agent`` log message like this
+
+    .. code-block:: shell-session
 
        level=info msg="Initializing ClusterMesh routing" path=/var/lib/cilium/clustermesh/ subsys=daemon
```

Comment:
:+1: would be good if we could validate the same via `cilium status` (first, in-pod but ideally from the clusterwide CLI). Maybe if this is not available, file an issue for it?